apps/condo/domains/billing/tasks/sendBillingReceiptsAddedNotifications.js

```diff
@@ -0,0 +1,223 @@
+const { get, map, isEmpty, isUndefined } = require('lodash')
+
+const conf = require('@core/config')
+const { getSchemaCtx } = require('@core/keystone/schema')
+
+const { getStartDates, getLoadByChunks, getUniqueByField, getConnectionsMapping } = require('@app/condo/bin/lib/helpers')
```

Comment:
!! wrf !! you cant use bin utils inside the DOMAIN TASK LOGIC!

Reply:
moved that